samples/wifi/shell/prj.conf

```diff
@@ -74,6 +74,7 @@ CONFIG_LOG=y
 CONFIG_PRINTK=y
 CONFIG_SHELL=y
 CONFIG_SHELL_GETOPT=y
+CONFIG_SHELL_ARGC_MAX=30
```

Comment:
this is applicable for all samples that enable Wi-Fi shell, so, may be we should fix this in the `subsys/net/l2/wifi/Kconfig` ?